### DISCUSSION TITLE
ungoogled chromium still safe?

### DISCUSSION THREAD

**Discussion by Devious3901** [Eloston/ungoogled-chromium#2568] [Q&A]
Hi, today I read the Cybersecurity "Bible" from https://anonymousplanet.org/
And as a user of ungoogled-chromium (flatpak), I was surprised that they do NOT recommend using this browser.
https://anonymousplanet.org/guide.html#ungoogled-chromium
they also refer to a report from 2021 https://qua3k.github.io/ungoogled/

Also, I know that this browser has lower scores than Brave (although in my opinion it is no better, because it also has some backdoors, and recently there was information that it downloaded its VPN without users' permission)
https://privacytests.org/

So, I want to hear the answer from the developers who work on the browser, are we really safe, do security patches come on time, is it safe to use it on a flash drive, and why is Brave overhyped and sucks?

**Comment by networkException** [comment]
Most of the reported issues are outdated or highly platform dependent. Flatpak is generally well maintained, so are the upstream patch releases (they generally follow on the same day or day after the chromium stable release).

As for this privacy testing thing, ungoogled-chromium does NOT aim to adblock or filter anything out of the box, unlike other browsers. It just removes internal requests to Google, among a few other tweaks. For privacy, use uBlock Origin.

**Reply by PF4Public** [reply]
> For privacy, use uBlock Origin.

In some cases uBlock Origin alone might not suffice. I would recommend doing one's own research on privacy. Although one should first decide on the degree of privacy one desires. There is no one-size-fits-all solution.
